tests/test_ucm.py

```diff
@@ -282,8 +282,6 @@ def test_bad_args(self):
             'workspace_dir': self.workspace_dir,
             'results_suffix': 'test_suffix',
             't_ref': 35.0,
-            't_obs_raster_path': os.path.join(
-                REGRESSION_DATA, "Tair_Sept.tif"),

```

Comment:
Why is this key pair being removed from the test?

Reply:
`t_obs_raster_path` doesn't exist in the spec for UCM, and with the changes I made to validation, it now causes a key error when there are extra keys. I'm guessing this was just outdated

Reply:
oh interesting! Thanks.